src/components/modal-design-library/editor.scss

```
@@ -57,7 +61,7 @@
 		display: flex;
 		justify-content: flex-end;
 		padding: 0 24px;
-		height: 80px;
+		height: 50px;
```

I suggest bringing this back because the toolbar became smaller
```suggestion
		height: 80px;
```